lib/hotwire_combobox/helper.rb

```diff
@@ -122,7 +122,30 @@ def hw_call_method_or_proc(object, method_or_proc)
         if method_or_proc.is_a? Proc
           method_or_proc.call object
         else
-          object.public_send method_or_proc
+          hw_call_method object, method_or_proc
+        end
+      end
+
+      def hw_call_method(object, method)
+        if object.respond_to? method
+          object.public_send method
+        else
+          raise NoMethodError, <<~MSG
+            Message from HotwireCombobox: [ACTION NEEDED]
+
+            `#{object.class}` does not respond to `##{method}`.
+
+            This method is used to determine how this option should appear in the combobox options list.
```

Comment:
If this method is used for other things than `to_combobox_display`, this error message might not match up?

Reply:
Yeah, I was thinking `#to_combobox_display` is the only case where the library expects you to implement a method other than `#id`. If this point is reached with any other method it'd be something the user provided. Which is supported but not documented.

Still, I agree ideally we'd always be accurate in this message. Maybe we can special-case the `#to_combobox_display` method since it should be the most common and return something more generic for anything else.

Reply:
Since just this line is unique to that, could just include it conditionally.

Also could pass the error message in as an argument or have a hash with messages for each method name and a fallback for anything else. Those may be overkill for now though.

Reply:
Yeah went with a one-off conditional for now. We can refine down the line if needed.